pages/index.js

```diff
@@ -19,6 +19,8 @@ export default function Home() {
 					rel='icon'
 					href='/favicon.ico'
 				/>
+			<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"></link>
```

Comment:
You should never use it this way. Usually you should either use react icons or font awesome icons in Nextjs.